USE-CASES.md

```diff
@@ -180,6 +183,77 @@ Such questions are answerable if the defender has the requisite “capabilities,
 
 This section lists the capabilities that we have identified as useful for our key use cases. (Think of these as the requirements we have for our key use cases, deferring implementation opportunities or challenges to a subsequent design proposal)
 
+#### Recognize whether the same device is seen again in the context of the same identity
+This applies to both within the same domain or across domains. Detecting a same device returning would traditionally happen through storing IDs persistently in the browser and reusing them when a device is seen again. The lifespan of detection needs to be at a minimum 7 days.
+
+##### Use cases
+Account Creation: Seeing an identity being used with a device it has never been seen with before (or that's very different from the devices it has been seen with before) can indicate identity theft where a fraudster uses a known identity to open new accounts.
+Account Takeover: An account that is taken over will be accessed from a new device, i.e. a device it hasn’t been used with before. This can be treated as an additional risk compared to devices that have been regularly seen with said account. The lifespan during which a device can be recognized as ‘seen again’ is very relevant here, as shorter windows will make every account access look as if it came from a never-before-seen device.
+1st Party Fraud: A person committing 1st party fraud and then claiming to be a victim of identity theft can in some cases be detected by recognizing that their own device was used to commit said fraud.
+
+####  Recognize whether the same device is seen again in the context of multiple identities
+Similar to the above, but recognizing the usage of the same device with different identities
+
+##### Use cases
+Account Creation: Fraudsters often steal multiple identities. Seeing many different identities’ data coming from the same device is a high risk signal, since legitimate users mostly use a single identity with their device.
+Account Takeover: A similar point to the above holds for account takeover when e.g. a fraudster is using a list of leaked usernames and passwords to takeover multiple victims' accounts. Recognizing that these logins into multiple accounts are happening from the same device can be used to indicate high risk.
+
+#### Retrieve a device’s IP address
```

Comment:
Would it be possible to tease these apart a bit? IP address is one signal that's potentially giving us these capabilities:

+ Build and maintain a reputation of networks connecting to your system (datacenters, geo-hopping proxies) (from VPN/TOR detection)
+ Distinguish client endpoints for rate limiting (from IP for account creation)
+ Distinguish client endpoints for resource binding (from IP for account takeover)
+ Distinguish client endpoints for blocking / preventing repeat abuse
+ Approximate geo location

Reply:
I recommend we discuss these points with the author of the original capabilities submission or else review these comments in a future meeting.

Reply:
Hi Philip, 

Thank you for your suggestion. I feel that
- Distinguish client endpoints for rate limiting (from IP for account creation)
- Distinguish client endpoints for resource binding (from IP for account takeover)
- Distinguish client endpoints for blocking / preventing repeat abuse
are basically one capability "Distinguish client endpoints" with three use cases. 

If we reduce these bullets to the single capability "Distinguish client endpoints", then I think we're missing the utility of the IP block nature in your outline by talking about networks and client endpoints. Semantically I'm interpreting the client endpoint as the machine being used, and the network as corresponding to a specific IP block. And I consider IP the link between the two. In this case, I feel that we need to add the capability:

- Know a client endpoint's network and which client endpoints belong to a network

Additionally however, I have a concern about the vagueness of the term "networks" as it doesn't have any granularity specified to it. A network could be a small CIDR block or a large mobile provider's network. What constitutes a network can also have a variable interpretation both depending on use case and on implementation of the network. Therefore I think there are risks in defining upfront what "network" means, as opposed to having a capability specifically for IP addresses.

Reply:
I'd actually veer away from "distinguish client endpoints" and focus on the second half of those descriptions (rate limiting, resource binding, blocking) as the specific things we need to enable.

Are there aspects of a network (ASN? first 2 octets?) that would be helpful for a proxy to forward? 
If a proxy provided an oracle that took sets of exit IPs and answered questions (e.g. How many distinct IpOrgs are these clients from), are there particularly helpful questions you would ask it?

Reply:
I'm sure Yarne will have a lot more to say, but here's a use case for you - today there are companies that sell IP Intelligence solutions, including whether or not IPs are associated with anonymizer networks, proxies, and TOR.  Behind the scenes, providers of this intelligence are often employing teams of workers to sign up for various commercial/criminal proxy services and map out IP ranges for identification (the data providers can typically distinguish these services from corporate VPNs or other similar networks).  It is hugely valuable intelligence to eCommerce and financial platforms, as fraud rates for users with these characteristics tend to be sky high. Do you see an alternative means of gathering this intelligence if some form of IP blindness is enabled?  A similar scenario exists where platforms take it upon themselves to identify IP ranges tied to fraud attacks. IP is a critical component for identifying participants in an attack, and ultimately stopping it as quickly as possible. What's an alternative that a criminal organization couldn't easily bypass?

Reply:
On an identity fraud level It is actually very important for us to be able to distinguish client endpoints. Fraud can happen from individual endpoints. Some fraudsters can also even be located closely to their victims, in which case bucketed information would be too coarse.
Bucketing client endpoints together has major problems, as one endpoint engaged in fraud would either negatively impact everyone else in that bucket by default or would be harder to detect. Also, just using ASN or the first 2 octets is very broad (first 2 octets would bundle about 32,000 IPs together).

Reply:
Per the conversation on the last call, and these comments, I am going to add the following capabilities and leave the rest entact:
* Build and maintain a reputation of networks connecting to your system (datacenters, geo-hopping proxies) (from VPN/TOR detection)
* Distinguish client endpoints for rate limiting (from IP for account creation)
* Distinguish client endpoints for resource binding (from IP for account takeover)
* Distinguish client endpoints for blocking / preventing repeat abuse
* Approximate geo location
* Know a client endpoint's network and which client endpoints belong to a network

Reply:
Please see the additions starting on line 205.

Reply:
Marking resolved so that we can complete the merge. Please suggest further changes if you are not satisfied with the result.